### DISCUSSION TITLE
primefaces11.0.0-jakarta DF not work

### DISCUSSION THREAD

**Discussion by jaejong** [primefaces/community#237] [PrimeFaces] [answered]
Hellow!
with primefaces11.0.0-jakarta 
PrimeFaces.current().dialog().openDynamic("/viewer", getCommonOptions(), null);
not work with no error messages.

PrimeFaces class packaged in primefaces11.0.0-jakarta.jar import  javax instead of jakarta.
How can I fix it?

Thanks.

**Comment by FlipWarthog** [comment, marked as answer]
Is the issue present in 13.0.2? Or, can you create a reproducer using the PrimeFaces test project? There is a branch set up specifically for Jakarta.

https://github.com/primefaces/primefaces-test

**Comment by jaejong** [comment]
I solved this.
When I upgrade from 6.0.0 to 11.0.0, I comented follows:
       
==
	<action-listener>org.primefaces.application.DialogActionListener</action-listener>
	<navigation-handler>org.primefaces.application.DialogNavigationHandler</navigation-handler>
	<view-handler>org.primefaces.application.DialogViewHandler</view-handler>
 ==
But it is also required this configuration at version 11.0.0.
